code/apps/munki-notifier/munki-notifier/da.lproj/InfoPlist.strings

```diff
@@ -1,3 +1,3 @@
-/* (No Commment) */
```

Comment:
This is interesting because something (one of the tools that generates localized files) is actually generating that typo. We can fix it, but it will most likely return.

Reply:
Yes, I see `(No Commment)` elsewhere on GitHub too. I'll try to find the typo upstream.